src/react-apps/applications/dashboard/src/dashboardServices/serviceOverview/servicesOverview.tsx

```diff
@@ -77,6 +77,12 @@ const getListOfDistinctServiceOwners = (services: any, currentUser?: string) =>
   return allDistinctServiceOwners;
 };
 
+const getListOfServicesExcludingCodelist = (services: any) => {
+  return services.filter((service: any) => {
```

Comment:
return services.filter((service: any) => {
    return service.name !== 'codelists';
  });

Can be written as:

  return services.filter((service: any) => service.name !== 'codelists');